src/box/sql/expr.c

```diff
@@ -3192,13 +3192,17 @@ expr_code_int(struct Parse *parse, struct Expr *expr, bool is_neg,
 		size_t len = strlen(z);
 		bool unused;
 		if (sql_atoi64(z, &value, &unused, len) != 0 ||
-		    (is_neg && (uint64_t) value > (uint64_t) INT64_MAX + 1)) {
+		    (is_neg && (uint64_t)value > (uint64_t)INT64_MAX + 1)) {
 			diag_set(ClientError, ER_INT_LITERAL_MAX, sign, z);
 			parse->is_aborted = true;
 			return;
 		}
 	}
-	if (is_neg)
+
+	/* We don't need to negate INT64_MIN value because it's
+	   negation is equal to it.
+	*/
```

Comment:
Please move `*/` to the previous line. You can also make the first line a little longer, up to 80 characters.

Reply:
Sorry, this suggestion seems to be wrong.

Reply:
Multiline comments are started from the `/*` (or `/**`) line.

Correct:

```
/*
 * Foo.
 * 
 * Bar. Baz. Fiz.
 */
```

Wrong:

```
/* Foo.
 * 
 * Bar. Baz. Fiz.
 */
```

https://lkml.org/lkml/2016/7/8/625

Reply:
@Totktonada 
Fixed